packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -131,8 +131,8 @@ export class ColumnHeaderCell extends React.Component<IColumnHeaderCellProps, IC
     };
 
     public render() {
-        const { isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
-        const classes = classNames(Classes.TABLE_HEADER, {
+        const { className, isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
+        const classes = classNames(className, Classes.TABLE_HEADER, {
```

Comment:
Gotcha. Do you want me to update RowHeaderCell as well? Thats where I got the convention.